beacon_node/beacon_processor/src/lib.rs

```diff
@@ -1209,33 +824,29 @@ impl<T: BeaconChainTypes> BeaconProcessor<T> {
                     .as_ref()
                     .map_or(false, |event| event.drop_during_sync);
 
+                let idle_tx = idle_tx.clone();
                 match work_event {
                     // There is no new work event, but we are able to spawn a new worker.
                     //
                     // We don't check the `work.drop_during_sync` here. We assume that if it made
                     // it into the queue at any point then we should process it.
                     None if can_spawn => {
-                        let toolbox = Toolbox {
-                            idle_tx: idle_tx.clone(),
-                            work_reprocessing_tx: work_reprocessing_tx.clone(),
```

Comment:
The `beacon_node/network` crate has it's own `work_reprocessing_tx` so there's no need to pass it to each worker. This means we no longer need the `Toolbox` struct (since it's only a single-item struct now).